.github/scripts/codespell.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+SCAN_RESULT=`codespell -H --quiet-level=3 --skip="*.h,*.cmake,*.c,*.hpp,*.cpp,*.sh,*.py,test/supp/*.supp,./.venv" --ignore-words-list="ASSER,Tne,ba,BA"`
```

Comment:
If this is just one line to trigger, I don't see why it has to be a separate script instead of having the command directly in the workflow `run` command.
If it were to be used by developers apart from CI, it should be a python script so it could be run both on Windows and Linux.

Reply:
We decided with @lplewa that the script is good, can be re-used.

As for Windows, is it supported? if so, perhaps we should write that script in python...

Reply:
It's a `pip` package, it should work on Windows as well.